test/unit_tests/aie/21_shim_dma_packet/aie.mlir

```diff
@@ -22,7 +22,7 @@ module @kernel_gemm  {
   %4 = aie.switchbox(%3)  {
     %43 = aie.amsel<0> (0)
     %44 = aie.masterset(West : 0, %43)
-    aie.packet_rules(DMA : 0)  {
+    aie.packet_rules(South : 0)  {
```

Comment:
shim tile itself has no DMA port, using shimmux for a complete design, leave it for now